meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,221 @@
+
+⚠️ Note the size of the vectors must be the same across all of your documents. If the sizes are inconsistent you will see an error in your tasks.
+
+```bash
+curl -X POST -H 'content-type: application/json' \
+'localhost:7700/indexes/myvectors/documents' \
+--data-binary '[
+    { "id": 0, "_vectors": [0, 0.8, -0.2], "text": "Hello World" },
+    { "id": 1, "_vectors": [1, -0.2, 0], "text": "Hello Doggy" },
+    { "id": 2, "_vectors": [[0.5, 3, 1], [-0.2, 4, 6]], "text": "Hello People" }
+]'
+```
+
+### Query Meilisearch using Vectors
+
+You can query the nearest documents using the default `/search` route by sending the vector of your query via the `vector` field. Again, for the first iteration you must compute the vector query on your side before requesting from Meilisearch.
+
+```bash
+curl -X POST -H 'content-type: application/json' \
+'localhost:7700/indexes/myvectors/search' \
+--data-binary '{ "vector": [0, 1, 2] }'
+```
+
+### Similarity score
+As part of v1.3.0 we will return a similarlity score as part of the documents retrieved. This score will be between 0 and 1 and viewed under the `_semanticSimilarity` property. The higher the ranking score, the more similar the document.
+
+```json
+{
+  "hits": [
+    { "id": 0, "_vectors": [0, 0.8, -0.2], "text": "Hello World", "_semanticSimilarity": 0.6754 },
+    { "id": 1, "_vectors": [1, -0.2, 0], "text": "Hello Doggy", "_semanticSimilarity": 0.7546 },
+    { "id": 2, "_vectors": [[0.5, 3, 1], [-0.2, 4, 6]], "text": "Hello People", "_semanticSimilarity": 0.78 }
+  ],
+  "query": "",
+  "vector": [0, 1, 2],
+  "processingTimeMs": 0,
+  "limit": 20,
+  "offset": 0,
+  "estimatedTotalHits": 2
+}
+```
+
+TBD - should this be with the new endpoint instead??
+When launching Meilisearch, use the following flag:
+```bash
+meilisearch --experimental-score-details
+```
+
+🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/meilisearch/discussions/621).
+
+⚠️ The experimental feature API can break between Meilisearch versions. Be careful if using it in production.
+
+Done by @Kerollmops in #3825
+
+
+## Display ranking details at search
+
+Meilisearch v1.3.0 introduces score details per document via the `showRankingScore` search query parameter. With this in place each document will return with a `_rankingScore` property displaying a score between 0 and 1. The higher the ranking score, the more relevant the document.
+
+```json 
+"_rankingScore": 0.8575757575757575,
+```
+
+Additionally, as an experimental feature, you can view detailed scoring per ranking rule for each document via the `showRankingScoreDetails` search query parameter. With this in place each document will return with a `_rankingScoreDetails` JSON object. That object will contain a property and a score value for each ranking rule. 
```

Comment:
I would do another sub-section as `experimental` and quickly say how to activate it.

Reply:
good idea, thats changed now